WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -292,7 +295,7 @@ def execute(self):
       self.log.warn('Job has no CPU requirement defined in JDL parameters')
 
     # Job requirement for a number of processors
-    processors = int(params.get('NumberOfProcessors', 1))
+    processors = int(params.get('NumberOfJobProcessors', 1))
```

Comment:
This is a job JDL parameter which is NumberOfProcessors, you can not change the name here. The NumberOfJobProcessors that you assign in the Computing Element is not the same thing.

Reply:
OK fixed.